### ISSUE TITLE
Questions about training hyperparameters

### ISSUE THREAD

**Issue by dongshenhan** [xavhl/GlassSemNet#1]
We did not find an explanation of the relevant training hyperparameters, including learning rate, number of batches, etc. Please help us to point it out, thanks. It is very important for us.
